arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -171,7 +172,10 @@ def start_pcp(
                     error.cmd[0], error.returncode, error.output
                 )
             )
-        return "success", PerfOutput(pcp_out_json)
+        pcp_metrics_list = pcp_out_json["@pcp"]["@hosts"][0]["@metrics"]
```

Comment:
Is there ever a case where only getting the first host could be a limitation?

Reply:
I don't believe so with the current design. Multiple hosts in the output should require PCP itself to be configured to collect data from remote agents. As the plugin stands, we explicitly only collect data from the local node/pod, so there should only ever be a single `host` element in the output.